app/helpers/application_helper.rb

```diff
@@ -469,8 +469,6 @@ def link_to_content_update(text, url_params = {}, html_options = {})
 
   def password_complexity_requirements
     rules = OpenProject::Passwords::Evaluator.rules_description
-    # use 0..0, so this doesn't fail if rules is an empty string
-    rules[0] = rules[0..0].upcase
```

Comment:
This in-place modification seems to have been used to title-ize the `rules_description`. However, `rules_description` already returns a string that originates from `I18n`, so all necessary titleization should already have been applied on the locale-level.